src/index.ts

```diff
@@ -230,4 +230,8 @@ export default class Api {
   public setAccount(account: string): void {
     this.config.settings.account = account;
   }
+
+  public setDeviceToken(deviceFcmToken: string): void {
```

Comment:
I would add getter too, `getDeviceToken` and use it instead of directly `this.config.settings.deviceFcmToken`

Reply:
I have added getter